src/Symfony/Component/Yaml/Dumper.php

```diff
@@ -30,6 +30,10 @@ class Dumper
      */
     public function __construct($indentation = 4)
     {
+        if ($indentation < 1) {
+            throw new \InvalidArgumentException('The indentation must be greater than zero.');
```

Comment:
The docblock states that the argument must be an integer and we don't do scalar type checks in Symfony.